mastodon-well-known-host-meta

```diff
@@ -0,0 +1,9 @@
+---
+layout: none
+permalink: .well-known/host-meta
+---
+<xml version="1.0" encoding="UTF-8"?>
+<XRD xmlns="http://docs.oasis-open.org/ns/xri/xrd-1.0">
+  <Link rel="lrdd" template="https://some.hacklab.fi/.well-known/webfinger?resource={uri}" type="application/xrd+xml" />
```

Comment:
According to https://gist.github.com/junosuarez/012755df0243ad17d942ddf26eb707a7#theory

> if we're ok using the same forwarding information for any account on the domain, we can ignore the resource query param and use a static json document as our response!

So this line should likely be: `<Link rel="lrdd" template="https://cool.example/.well-known/webfinger" type="application/xrd+xml" />`